src/components/svgs/nodes/misc-nodes.ts

```diff
@@ -16,6 +16,7 @@ import jrEastLineBadge from './jr-east-line-badge';
 import facilities from './facilities';
 import text from './text';
 import i18nText from './i18n-text';
+import qingdaoMetroNumLineBadge from './qingdao-metro-num-line-badge';
```

Comment:
This should be between `jrEastLineBadge` and `facilities`. Just personal preference :)